frontend/src/Monitor/Leaderboard/Charts/ResourceProduction.tsx

```diff
@@ -85,8 +86,8 @@ export const ResourceProductionChart: React.FC<LeaderboardChartProps> = ({data,
             <div className='leaderboardChartForm'>
                 <FormControl className={classes.formControl}>
                     <Select value={mode} onChange={onModeChange}>
-                        <MenuItem value='avg'>Average Resource Production</MenuItem>
-                        <MenuItem value='sum'>Total Resource Production</MenuItem>
+                        <MenuItem value='avg'>Avg. Res. Prod.</MenuItem>
+                        <MenuItem value='sum'>Total Res. Prod.</MenuItem>
```

Comment:
Seeing as the chart title is already "Resource Production" we can just change these to "Average"/"Avg" and "Total"